packages/compile-solidity/compilerSupplier/loadingStrategies/Native.js

```diff
@@ -1,31 +1,26 @@
-const { execSync } = require("child_process");
-const LoadingStrategy = require("./LoadingStrategy");
-const VersionRange = require("./VersionRange");
+const { execSync } = require('child_process');
```

Comment:
Why'd you change the strings? Can you please undo this? It'll just get overwritten by prettier next commit anyway